text/0000-roadmap-2020.md

```diff
@@ -0,0 +1,368 @@
+* Our organization and Rust teams have grown.
+
+This is great news! But with this growth comes challenges.  We are
+finding that it is harder and harder to ensure communication across
+the organization. It can often be challenging to find enough people to
+do the work we would like to get done, which in turn leads to burnout
+or people leaving the project. We've identified three major goals that
+we think will help.
+
+### Improve visibility into the state of initiatives and design efforts
+
+Right now it is very difficult to answer questions like "what are the
+active efforts and how can I help" to "what is the status of feature
+X". This is true both for folks who are deeply embedded in the Rust
+organization and for newcomers.
+
+There are many ways to improve visibiility, but the most basic step is
+simply expending more effort on posting updates and documenting the
+status.  Things like the Inside Rust blog are helpful here, and we
+should look for other ways to incorporate lightweight status updates
+into our workflows.
+
+### Increase mentoring, leadership, and organizational bandwidth
+
+One common challenge for us is that we seem to lack enough people to
+get the work done that we would like to get done. But what we're
+missing is not just *any* people, it's people who can help to do the
+"leadership" work that knits the project together.
+
+This work takes many forms. Sometimes it is technical, such as writing
+mentoring instructions on issues, but more often it is organizational,
+such as running meetings, posting blog posts (see the previous point),
+or making plans.
+
+We have made great progress over the years by intentionally focusing
+on the "on-ramp" to contribution, through efforts like tagging E-easy
+issues. We've made more limited progress on helping people "step up"
+towards leadership roles.
+
+Part of the problem here is money. One of the biggest challenges
+around organizational work is that it is quite demanding in terms of
+time. It requires availability. It is difficult to do in your spare
+time. Therefore, helping to ensure that it is easier for people to get
+paid for their work on Rust -- and especialy their **organizational**
+work -- is one way we might make progress here.
+
+However, it's worth emphasizing that this doesn't necessarily mean
+people whose job description is *solely* to work on Rust. There are
+many companies using Rust, and many of them would like to help out,
+but we need to do better at harnessing and directing those efforts.
+As Parity put it in their #rust2020 post:
+
+> “We, too, have team members who are interested in helping on
+> specialization or fixing the aforementioned bugs. However, it’s
+> often unclear whether the work is worthwhile. To a business, it is
+> hard to argue that one might spend a month or two working on a new
+> feature without any assurance that the approach taken would be
+> accepted.”
+>
+> -- [Benjamin Kampmann, speaking for Parity](https://www.parity.io/rust-/)
```

Comment:
This link is broken.

Reply:
Should be https://www.parity.io/rust-2020/

Reply:
```suggestion
> -- [Benjamin Kampmann, speaking for Parity](https://www.parity.io/rust-2020/)
```